_shared/project/docker-compose.yml

```diff
@@ -4,6 +4,8 @@ services:
     image: postgres:{{ cookiecutter.get("__postgres_version", "11.5-alpine") }}
     ports:
       - "127.0.0.1:{{ cookiecutter['__postgres_port'] }}:5432"
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
Via is going to use the latest version of Postgres, not the old version that our other apps are still on. The latest version requires this envvar otherwise it refuses to start up